CesiumGeometry/include/CesiumGeometry/Axes.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "CesiumGeometry/Library.h"
+#include <glm/mat3x3.hpp>
+
+namespace CesiumGeometry {
+
+/**
+ * @brief Coordinate system conversion matrices
+ */
+struct CESIUMGEOMETRY_API Axes final {
+
+  /**
+   * @brief A matrix to convert from y-up to z-up orientation,
+   * by rotating about PI/2 around the x-axis
+   */
+  static const glm::dmat4 Y_UP_TO_Z_UP;
```

Comment:
those matrices are good candidates for unit tests too. We can test it out by multiply a Y-vector with the matrix to see if it converts to Z-axis or something similar like that

Reply:
One problem is that "converting from one up-vector to another" can be done in different ways. Specifically (and maybe counterintuitively), 

    Y_AXIS * Y_UP_TO_Z_UP == -Z_AXIS 

Note that it's **MINUS** the `Z_AXIS`. This wouldn't necessarily be the case for other ways of "making the Z-axis point 'up'". 

That's why I *explicitly* added the *"...by rotating about PI/2 around the x-axis"* part in the comment - without that, the matrix could have many different forms, all achieving the result of Z pointing 'up'.

However, of course you're right regarding the unit test, and I just added them. Considering that the matrix columns are just the images of the unit vectors, these tests could be considered as a very elaborate form of saying 

    assert(X_TO_Z.elements == { 1, 0, 0, 0, 0, 0, 1, 0, 0, -1, 0, 0, 0, 0, 0, 1 })

but having that encoded as tests isn't wrong either.